views/add-ons.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Show available add-ons from WP101.
+ *
+ * @global $api       An instance of WP101\API;
+ * @global $addons    An array of add-ons from the WP101 API.
+ * @global $purchased Slugs of any Series this site already has access to.
+ *
+ * @package WP101
+ */
+
+use WP101\TemplateTags as TemplateTags;
+
+?>
+
+<main class="wrap wp101-addons">
+	<h1>
+		<?php echo esc_html( _x( 'WP101 Add-ons', 'listings page title', 'wp101' ) ); ?>
+	</h1>
+
+	<?php settings_errors(); ?>
+
+	<?php if ( empty( $addons['addons'] ) ) : ?>
+
+		<div class="notice notice-warning">
+			<p><?php esc_html_e( 'There are no add-ons currently available for WP101!', 'wp101' ); ?></p>
+		</div>
+
+	<?php else : ?>
+
+		<p><?php esc_html_e( 'Enhance your WP101 experience with these add-ons:', 'wp101' ); ?></p>
+
+		<div class="wp101-addon-list">
+			<?php foreach ( $addons['addons'] as $addon ) : ?>
```

Comment:
Need to confirm `$addons['addons']` exist before using in the foreach.

Reply:
This check is performed one line 23:

```php
<?php if ( empty( $addons['addons'] ) ) : ?>

  <div class="notice notice-warning">
    <p><?php esc_html_e( 'There are no add-ons currently available for WP101!', 'wp101' ); ?></p>
  </div>

<?php else : ?>

  <p><?php esc_html_e( 'Enhance your WP101 experience with these add-ons:', 'wp101' ); ?></p>

  <!-- ... -->

<?php endif; ?>
```